src/nvidia_ops_manager.py

```diff
@@ -65,17 +66,11 @@ class NvidiaOpsManagerUbuntu(NvidiaOpsManagerBase):
 
     OS_RELEASE = os_release()
 
-    def __init__(self):
+    def __init__(self, driver_package: str = ""):
+        self._driver_package = driver_package if driver_package else "cuda-drivers"
```

Comment:
I do not think it is necessary to check the default value here. It is set in the `config.yaml ` already.